pkg/metrics/types/types.go

```diff
@@ -102,6 +105,36 @@ type InstallationFailed struct {
 	GenericEvent `json:",inline"`
 }
 
+// UpgradeStarted event is sent back home when an upgrade starts.
+type UpgradeStarted struct {
```

Comment:
It looks like these events are missing InitialVersion and TargetVersion.

https://github.com/replicatedhq/kots/blob/944526c4d46d0dd301041f957c9a7d1f796cbd1b/pkg/embeddedcluster/metrics.go#L16-L36